Docker-compose/docker-compose.yml

```diff
@@ -107,9 +103,9 @@ services:
       - SATOSA_UI_PRIVACY_URL_EN=${SATOSA_UI_PRIVACY_URL_EN:-https://example_organization.org/privacy}
       - SATOSA_UI_PRIVACY_URL_IT=${SATOSA_UI_PRIVACY_URL_IT:-https://example_organization.org/it/privacy}
       - SATOSA_USER_ID_HASH_SALT=${SATOSA_USER_ID_HASH_SALT:-CHANGE_ME!}
-      - SATOSA_REQUESTED_ATTRIBUTES=${SATOSA_REQUESTED_ATTRIBUTES:-[]}
 
-      - GET_IDEM_MDQ_KEY=${GET_IDEM_MDQ_KEY:-true}
+      - SATOSA_REQUESTED_SPID_ATTRIBUTES=${SATOSA_REQUESTED_SPID_ATTRIBUTES:-[]}
+      - SATOSA_REQUESTED_CIE_ATTRIBUTES=${SATOSA_REQUESTED_CIE_ATTRIBUTES:-[]}
```

Comment:
@MdreW could you please add default values, considering the conventional spid or cie sets?